# AMRiSAUT_Data
This analysis repository contains data and code used in the study (Analysis of Methicillin Resistance in Staphylococcus Aureus Sepsis Using TDbasedUFE) by S. Watanabe and Y-h. Taguchi. 

---
## Index

- [**Code**](#code)
- [**Data_GSE220188**](#data_gse220188)
- [**Data_Rds**](./Data_Rds)
- [**Result_Figire**](./Result_Figure)
- [**Result_Table**](./Result_Table)

## Code

This folder is available for [**Code**](./Code).

The R source code (`.R`) and R Markdown (`.Rmd`) files used for the analysis are all stored here. 
The directory names in the source files need to be changed according to the user's folder hierarchy.
However, the file names remain unchanged, so please use search or other methods to reference files in different folders.

## Data_GSE220188

This folder is available for [**Data_GSE220188**](./Data_GSE220188).

The files directly obtained from [GSE220188](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE220188) and the spreadsheet edited for analysis are included here.
For the analysis, `Motif-In-Peaks-Summary.rds` included in [GSE220188](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE220188) is required. Please obtain it directly from [Gene Expression Omnibus](https://www.ncbi.nlm.nih.gov/geo/) for each dataset ([Click here](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE220188)).

## Data_Rds


